dev/workflow/setup_expt.py

```diff
@@ -56,7 +56,7 @@ def _update_defaults(dict_in: dict) -> dict:
     # Combine host.info and inputs_dict into a single dict, add some additional keys
     host_plus_inputs_dict = AttrDict(host.info, **inputs_dict_remapped)
     host_plus_inputs_dict.HOMEgfs = _top
-    host_plus_inputs_dict.MACHINE = host.machine.upper()
+    host_plus_inputs_dict.MACHINE = str(host).upper()
```

Comment:
```suggestion
    host_plus_inputs_dict.MACHINE = host.upper()
```

Reply:
This doesn't work.  It looks for an `upper` method in the `Host` class, so `str()` is required first.